### PR TITLE
[HDInsight] Added support for ESP with AAD-DS

### PR DESCRIPTION
**Pull request by wawon-msft**
## Description
We made a change to the Swagger spec to add two new parameters in SecurityProfile for cluster creation. This PR contains the regenerated SDK (which was regenerated with a new API version), and re-recorded all tests. There were quite a few minor changes to the SDK due to the API version bump, but I expect this to contain breaking changes.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [ ] Please add REST spec PR link to the SDK PR
- [ ] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [ ] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [ ] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [ ] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

File: src/SDKs/HDInsight/Management.HDInsight/Generated/SdkInfo_HDInsightManagementClient.cs [hunk 1]
```diff
@@ -19,14 +19,14 @@ internal static partial class SdkInfo
...
               {
-                new Tuple<string, string, string>("HDInsight", "Applications", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "Clusters", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "Configurations", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "Extension", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "Locations", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "Operations", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "ScriptActions", "2015-03-01-preview"),
-                new Tuple<string, string, string>("HDInsight", "ScriptExecutionHistory", "2015-03-01-preview"),
+                new Tuple<string, string, string>("HDInsight", "Applications", "2018-06-01-preview"),
```
Comment (dsgouda) [inline src/SDKs/HDInsight/Management.HDInsight/Generated/SdkInfo_HDInsightManagementClient.cs]: For API version bumps, the nuget major version must be bumped

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good apart from versioning

File: src/SDKs/HDInsight/Management.HDInsight/Generated/SdkInfo_HDInsightManagementClient.cs (hunk 1 above)
Comment (wawon-msft) [reply src/SDKs/HDInsight/Management.HDInsight/Generated/SdkInfo_HDInsightManagementClient.cs]: Sounds good.

**Comment by wawon-msft** [comment]
I can squash my commits if everything else looks good.

**Review by dsgouda — APPROVED** [review]
LGTM, please squash the commits

**Comment by wawon-msft** [comment]
Apologies, I'm going to add one more commit, where I regenerate with these Swagger changes: https://github.com/Azure/azure-rest-api-specs/pull/4163

I missed these changes earlier, and these changes need to go together with this PR.

**Review by dsgouda — APPROVED** [review]
LGTM
@wawon-msft is this good to merge?

**Pull request merged**